billing_inc.php

```diff
@@ -268,6 +268,8 @@
 	</tr>
 
 <?php
+	uksort ( $t_bugnote_stats['users'], function ($a, $b) { return strcmp(user_get_username($a), user_get_username($b)); } ); // _cis_
```

Comment:
Please follow our [coding guidelines](https://www.mantisbt.org/wiki/doku.php/mantisbt:coding_guidelines)
Break the long line.